src/game/logic/object/locomotor.cpp

```diff
@@ -1174,13 +1231,188 @@ void Locomotor::Move_Towards_Position_Hover(
     }
 }
 
+bool Within_Epsilon(float f)
+{
+    return GameMath::Fabs(f) < 0.001f;
```

Comment:
use GAMEMATH_EPSILON

Reply:
GAMEMATH_EPSILON is 0.0001f, this is 0.001f. Would rather keep things matching stock to prevent hard-to-debug issues later...